FastMenu/index.tsx

```diff
@@ -1,3 +1,9 @@
+/*
```

Comment:
No.

Reply:
To clarify beyond my kneejerk reaction, I do not have headers in my files, because they're a complete waste of space and add extra maintenance burden — case in point, it's not 2023 anymore and so the header of every file would need updating, which additionally interferes with github showing the latest commit. Plus, this file isn't even part of Vencord.

Reply:
I also forgot to remove this from testing as I threw it into my vencord plugins.